README.md

```diff
@@ -96,7 +96,47 @@ not be repeated, and should be placed in a folder named by date.
 Repeated events are always shown at the top of the events page, and 
 do not expire.
 
-### 4. How do I add a page redirect?
+### 4. How do I add a community document?
+
+A community document is a living document that represents a community effort
+to discuss a question or idea. We call it "living" because we encourage community
+members to edit and contribute to the documents over time, and so there isn't
+a "publish" date associated with it. For this reason, we use [the wiki](https://github.com/USRSE/usrse.github.io/wiki) associated
+with the repository here. 
+
+![assets/img/usrse-book-small.png](assets/img/usrse-book-small.png)
+
+If you ever need to clone this content, you can do:
+
+```bash
+$ git clone git@github.com/USRSE/usrse.github.io/wiki
```

Comment:
I can clone `https://github.com/USRSE/usrse.github.io.wiki.git` but that `git@` formulation "does not exist" for me. Should some caveat be added about organization membership or somesuch?

Reply:
ah, that's for ssl. We can definitely change to standard https (not everyone uses ssl, but someone that does will know to adjust the name!)

Reply:
```suggestion
$ git clone https://github.com/USRSE/usrse.github.io.wiki.git
```

Reply:
Would that work @mtbc?

Reply:
That works. :+1: Also, just so you know: I now find that if, in the other version, I change a couple of punctuation items to `git@github.com:USRSE/usrse.github.io.wiki` then that also works.